test/routers/handlers/individual-statement/individualStatement.int.ts

```diff
@@ -0,0 +1,73 @@
+import { VerificationStatement } from "@companieshouse/api-sdk-node/dist/services/psc-verification-link/types";
+import { HttpStatusCode } from "axios";
+import { parse } from "node-html-parser";
```

Comment:
This library node-html-parser is not currently used within Companies House.
Either we we can investigate getting this library approved or the test needs to be rewritten without it.